packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -6,6 +6,11 @@ import { ISemanticTextColors } from './ISemanticTextColors';
  * The collection of all semantic slots for colors used in themes.
  *
  * Note: text colors are defined in ISemanticTextColors.ts.
+ * The plan is to move completely to semantic slots and replace all usage of fabric palette slots.
+ * We still have a ways to go to complete this plan and we've been actively making some of these changes.
+ * At present, we're not adding any more slots to avoid bloating the already sizeable offering.
```

Comment:
We are still adding new slots, if necessary. Our policy hasn't changed.

Reply:
I have this to address your earlier comment "We should not add more palette slots (besides neutralSecondaryAlt)."

Reply:
Just reworded it along with some other changes to clarify!